client/js/common/direct-to-s3-upload.es6

```diff
@@ -28,7 +28,7 @@ const uploadFile = R.curry(input => {
       var url = `${presignedPost.s3_direct_url}`
       var file = input.files[0]
       var fileUrl = `${url}/tmp/${presignedPost.s3_uuid}/${file.name}`
-      var payload = R.merge(JSON.parse(presignedPost.s3_presigned_post), {file})
+      var payload = { ...JSON.parse(presignedPost.s3_presigned_post), ...{file} }
```

Comment:
Once again, which object's properties go first and which ones should go second?

Reply:
As far as I can tell from the docs (which don't go back to the version this is on) the second one is merged into the first / overwrites values on it
https://ramdajs.com/0.25.0/docs/#merge

this could be simplified more to this though:
```js
var payload = { ...JSON.parse(presignedPost.s3_presigned_post), file }
```